Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -943,19 +960,29 @@ define([
         // Replace: tile_colorBlend is 1.0 and the tile color is used
         // Mix: tile_colorBlend is between 0.0 and 1.0, causing the source color and tile color to mix
         var finalDiffuseFunction =
+            'bool isWhite(vec3 color) \n' +
```

Comment:
I added this because I noticed the `== vec3(1.0)` was too exact and started causing lots of aliasing on my desktop.